README.md

```diff
@@ -9,7 +9,7 @@ A curated, opinionated list of high-quality **remote** Model Context Protocol (M
 
 ## What is MCP?
 
-[Model Context Protocol](https://modelcontextprotocol.io/) (MCP) is a protocol that enables AI models to interact with external tools and resources. Remote MCP servers allow clients to securely connect to these services over the internet. 
```

Comment:
My editor removed the trailing newlines automatically. I'd leave that change in if it were my repo, but I'm happy to limit the diff to just the one line added in the table if preferred.